build.gradle.kts

```diff
@@ -32,8 +32,6 @@ dependencies {
 
   implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.6.0")
 
-  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.18.0")
```

Comment:
This is included in hmpps spring boot starter, and this version 2.18 has breaking changes

Reply:
it does indeed as we [found out on IRS api](https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/198/commits/89daeed7c487cdba983bafd78f7e444de3e34e90), but since tests had passed here we naturally assumed that there was a special thing going on in IRS! i mean surely there was json de/serialisation going on throughout testing…

Reply:
but the resource tests deal with json responses and there are plenty of assertions on responses 😕